startup/40-hdf5.py

```diff
@@ -196,8 +220,7 @@ def send_to_packing_queue(uid, data_type, froot=data_file_path.gpfs, move_first=
             print(f"scan {uid} was not successful.")
             return 
 
-    PilatusCBFHandler.froot = data_file_path[froot.name]
-    threading.Thread(target=pack_and_move, args=(data_type,uid,proc_path,move_first,)).start() 
+    threading.Thread(target=pack_and_process, args=(data_type,uid,proc_path,move_first,)).start() 
```

Comment:
According to the update below on line 242, the `move_first` is not an argument anymore.
```suggestion
    threading.Thread(target=pack_and_process, args=(data_type, uid, proc_path,)).start() 
```

Reply:
the code is now correct (no move_first argument)